proposed/http-message.md

```diff
@@ -1133,8 +1137,9 @@ interface StreamableInterface
      * @param int $length Read up to $length bytes from the object and return
      *     them. Fewer than $length bytes may be returned if underlying stream
      *     call returns fewer bytes.
-     * @return string|false Returns the data read from the stream, false if
-     *     unable to read or if an error occurs.
+     * @return string|null Returns the data read from the stream, or null if no bytes
```

Comment:
Please see the discussion; this was suggested by @mtdowling based on similar verbiage in python specs.